src/codec.ts

```diff
@@ -249,6 +249,25 @@ export class PGJsonb {
   }
 }
 
+/**
+ * @typedef PGOid
+ * @see Spanner.pgOid
+ */
+export class PGOid extends WrappedNumber {
+  value: string;
+  constructor(value: string) {
+    super();
+    this.value = value.toString();
```

Comment:
I thought this would work but it actually makes the behaviour different. I'm not sure I understand why. But this logic is the same as how we represent the `Int` type on line 144.